basepage/base_page.py

```diff
@@ -252,7 +252,10 @@ def enter_text(self, locator, text, with_click=True, with_clear=False, with_ente
             self.click(element)
 
         actions = ActionChains(self.driver)
-        actions.send_keys_to_element(element, text)
+        if 'explorer' in self.driver.name and "@" in str(text):
+            actions = self.handle_at_sign_for_ie(text, actions)
```

Comment:
Since the method is static, show that by calling it in a static way: `Basepage.handle....`